packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -901,6 +901,7 @@ async function handler(
     platformRescheduleUrl,
     platformCancelUrl,
     platformBookingUrl,
+    oneTimePassword: isConfirmedByDefault ? null : undefined,
```

Comment:
We need to update the one-time password after the booking, or we will get an undefined value in the email template. Currently, this is happening